### PR TITLE
Added more help documentation

### PR DESCRIPTION
**Pull request by sea5kg**

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -542,14 +545,29 @@ <h3>wsjcpp resources</h3>
...
 
-    <br><br>
-    <a class="anchorjs-link " name="wsjcpp_prepare" aria-label="Anchor" data-anchorjs-icon="#" style="padding-left: 0.375em;"></a>
-    <h3>wsjcpp prepare</h3>
-        Prepare some usefull files
+        <br><br>
+        <a class="anchorjs-link " name="wsjcpp_prepare" aria-label="Anchor" data-anchorjs-icon="#" style="padding-left: 0.375em;"></a>
+        <h3>wsjcpp prepare</h3>
+        
+        Preparing some useful files and packages for another systems
```
Comment (KeimaShikai) [inline web/index.html]: `Preparing some useful files and packages for other systems`

File: web/index.html [hunk 2]
```diff
@@ -542,14 +545,29 @@ <h3>wsjcpp resources</h3>
...
         <ul>
+            <li>wsjcpp prepare help - For more information</li>
             <li>wsjcpp prepare travis - Prepare base file (.travis.yml) for Travis CI</li>
             <li>wsjcpp prepare homebrew - Prepare package for homebrew</li>
+            <li>wsjcpp prepare dockerfile - Prepare sample of Dockerfile for build and publish current project</li>
```
Comment (KeimaShikai) [inline web/index.html]: `Prepare sample Dockerfile for build and publishing current project`

File: web/index.html [hunk 3]
```diff
@@ -542,14 +545,29 @@ <h3>wsjcpp resources</h3>
...
+            <li>wsjcpp prepare changelog - Prepare sample of CHANGELOG.md</li>
```
Comment (KeimaShikai) [inline web/index.html]: `Prepare sample CHANGELOG.md`

**Review by KeimaShikai — CHANGES_REQUESTED** [review]

File: web/index.html (hunk 1 above)
Comment (sea5kg) [reply web/index.html]: ok

File: web/index.html (hunk 3 above)
Comment (sea5kg) [reply web/index.html]: ok

**Review by KeimaShikai — APPROVED** [review]

**Pull request merged**